include/vapor/ParticleRenderer.h

```diff
@@ -59,7 +59,57 @@ class RENDER_API ParticleRenderer : public Renderer {
     virtual int _paintGL(bool fast);
 
 private:
+    struct {
+        size_t              ts;
+        int                 rLevel;
+        int                 cLevel;
+        std::vector<float>  tf_lut;
+        std::vector<double> tf_minMax;
+        VAPoR::CoordType    boxMin, boxMax;
+        float               radius;
+        bool                direction;
+        float               directionScale;
+        size_t              stride;
+        string              varName;
+        std::vector<std::string> fieldVars;
+    } _cacheParams;
+
+    struct _vertex {
+        glm::vec3  point;
+        float      value;
+    };
+
+    //std::vector<glm::vec4> _particles;
```

Comment:
Is there a specific reason to leave this comment line here?

Reply:
Nope.  Removed.